cupyx/fallback_mode/fallback.py

```diff
@@ -321,7 +321,12 @@ def _update_numpy_array(self):
                     if self._numpy_array is None:
                         self._numpy_array = base._numpy_array.view(type=_type)
                         self._numpy_array.shape = self._cupy_array.shape
-                        self._numpy_array.strides = self._cupy_array.strides
+                        # In some scenarios, the numpy array strides may be
```

Comment:
Can I have an example to reproduce this?

Reply:
Actually this error seems to be gone now, probably it was some kind of numpy/cupy incompatibility when I tested it the first time.

Reply:
Sorry, this is related to #8442, When running in a system with unified memory we get the following error

```FAILED tests/cupyx_tests/fallback_mode_tests/test_fallback.py::TestInplaceSpecialMethods::test_put_along_axis - ValueError: strides is not compatible with available memory
FAILED tests/cupyx_tests/fallback_mode_tests/test_fallback.py::TestArrayVariants::test_inplace - ValueError: strides is not compatible with available memory


 def _update_numpy_array(self):
        """
        Updates _numpy_array from _cupy_array.
        To be executed before calling numpy function.
    

        if self._supports_cupy:
            # cupy-compatible
            if base is None:
             _array)
            else:
                if not base._remember_numpy:

                        self._numpy_array.shape = self._cupy_array.shape
>                       self._numpy_array.strides = self._cupy_array.strides
E                       ValueError: strides is not compatible with available memory

cupyx/fallback_mode/fallback.py:324: ValueError
```

I think we can remove this fix from here and add it in the corresponding PR, or another option is to skip fallback mode tests when using the system allocator.

I discussed it a while ago with @leofang but is fallback mode being currently used? with the extremely high coverage that CuPy has, maybe it is worth deprecating it